src/CallHandler.tsx

```diff
@@ -59,8 +59,7 @@ import {MatrixClientPeg} from './MatrixClientPeg';
 import PlatformPeg from './PlatformPeg';
 import Modal from './Modal';
 import { _t } from './languageHandler';
-// @ts-ignore - XXX: tsc doesn't like this: our js-sdk imports are complex so this isn't surprising
-import Matrix from 'matrix-js-sdk';
+import Matrix from 'matrix-js-sdk/src/browser-index';
```

Comment:
this feels wrong - the browser-index shouldn't be mentioned in this capacity.

Reply:
If we're always referring to `src` now rather than the direct reference, wouldn't this have to be to `browser-index`, since if we refer to 'index' here won't that break things because the rest of rest-sdk will be importing browser-index?

Reply:
This would be the only place we reference `browser-index`, which isn't a safe export for us to rely upon. If we were importing `lib` below then it's very possible that all the builders got confused which is why the tsignore comment was needed. Now that the later imports are fixed to be `src` instead of `lib`, this import shouldn't have needed changes.

Reply:
`import Matrix from 'matrix-js-sdk'`: doesn't work because then the `MatrixCall` we get from `Matrix.createNewMatrixCall` is different to the `MatrixCall` we import from `matrix-js-sdk/src/webrtc/call`. We'd have to change the other imports to `matrix-js-sdk/src/webrtc/call` for them to be the same type. Or are you suggesting bypassing the root altogether and doing `import { createNewMatrixCall } from 'matrix-js-sdk/src/matrix'`?

Reply:
Done (for future reference, https://matrix.to/#/!bEWtlqtDwCLFIAKAcv:matrix.org/$DVj8lxlPoFyJTOOADc0bEn8hEdA0PfhLwPa0m2geZkA?via=matrix.org&via=mozilla.org&via=vector.modular.im )